aiogram/dispatcher/dispatcher.py

```diff
@@ -1221,3 +1222,35 @@ async def wrapped(*args, **kwargs):
             return wrapped
 
         return decorator
+
+    def bind_filter(self, callback: typing.Union[typing.Callable, AbstractFilter],
+                    validator: typing.Optional[typing.Callable] = None,
+                    event_handlers: typing.Optional[typing.List[Handler]] = None,
+                    exclude_event_handlers: typing.Optional[typing.Iterable[Handler]] = None):
+        """
+        Register filter
+
+        :param callback: callable or subclass of :obj:`AbstractFilter`
+        :param validator: custom validator.
+        :param event_handlers: list of instances of :obj:`Handler`
+        :param exclude_event_handlers: list of excluded event handlers (:obj:`Handler`)
+        """
+        self.filters_factory.bind(callback=callback, validator=validator, event_handlers=event_handlers,
+                                  exclude_event_handlers=exclude_event_handlers)
+
+    def unbind_filter(self, callback: typing.Union[typing.Callable, AbstractFilter]):
+        """
+        Unregister callback
```

Comment:
Invalid semantic. When you write "Register filter" abowe you will need to write here "Unregister filter" but not "callback"